integration-tests/micrometer-jmx/src/test/java/io/quarkus/it/micrometer/jmx/JmxMetricsRegistryTest.java

```diff
@@ -49,7 +49,7 @@ void testServerError() {
     }
 
     @Test
-    @Order(3)
+    @Order(4)
```

Comment:
if order is set to four, then who's getting 3?

Reply:
Just want to make sure it done on purpose.

Reply:
There were two 3s.... the test before this is also 3. It doesn't really matter that much.. the important bit is the deferral of the last one (10) so it can check prometheus results w/ all requests present.. but having 3 twice bugged me.

Reply:
when using @Order, we must be sure to not have multiple tests with the same order or the _ordering_ is not predictable. So, the change is absolutely right.